pepr.ts

```diff
@@ -46,6 +48,6 @@ import { prometheus } from "./src/pepr/prometheus";
     });
   }
 })().catch(err => {
-  Log.error(err);
+  Log.error(err, "Critical error during startup. Exiting...");
```

Comment:
I guess we missed this one and the above debug log in the previous logging PR, these should probably be using our new `setupLogger` child log stream?

Reply:
Updated to use a `startup` component.